manifests/mount.pp

```diff
@@ -1,30 +1,74 @@
-  $mount_method = $cvmfs::mount_method,
-  Optional[String] $cvmfs_external_fallback_proxy = undef,
-  Optional[String] $cvmfs_external_http_proxy = undef,
-  Optional[Integer] $cvmfs_external_timeout = undef,
-  Optional[Integer] $cvmfs_external_timeout_direct = undef,
-  Optional[String] $cvmfs_external_url = undef,
-  Optional[String[1]] $cvmfs_repository_tag = undef,
+# @summary
+#   Mount one cvmfs repository. Most parameters map as lower case
+#   versions of the raw cvmfs parameters.
+#
+# @see https://cvmfs.readthedocs.io/en/stable/apx-parameters.html CVMFS configuration parameters
+#
+# @example Trivial Mount
+#   cvmfs::mount{'files.example.org:
+#     cvmfs_server_url => 'http://web.exampple.org/cvmfs/files.example.org',
+#   }
+#
+# @example Mount with Environment Variables.
+#   cvmfs::mount{'foobar.example.org':
+#     cvmfs_env_variables => {
+#       'FOOT' => 'ball',
+#       'GOLF' => 'club',
+#     }
+#   }
+#
+# @param cvmfs_quota_limit Per mount quota limit, not relavent to shared cache. Sets cvmfs_quota_limit
+# @param cvmfs_server_url Stratum 1 list, typically `;` delimited. Sets CVMFS_SERVER_URL parameter.
+# @param cvmfs_http_proxy List of http proxies to use. Sets CVMFS_PROXY_LIST parameter.
+# @param cvmfs_timeout Sets CVMFS_HTTP_TIMEOUT parameter
+# @param cvmfs_timeout_direct Sets CVMFS_HTTP_TIMEOUT_DIRECT
+# @param cvmfs_nfiles Number of open files permitted on the OS. Sets CVMFS_NFILES
+# @param cvmfs_public_key Public key of repository, sets CVMFS_PUBLIC_KEYS
+# @param cvmfs_max_ttl Maximum effective TTL in seconds for DNS queries of proxy server names. Sets CVMFS_MAX_TTL
+# @param cvmfs_env_variables Sets per repo environments variables for magic links.
+# @param cvmfs_use_geoapi Set CVMFS_MAX_GEOAPI
+# @param mount_method Should the mount attempt be made with autofs or tranditional fstab mount. Do no use this.
+# @param cvmfs_repo_list If true the repository will added to the list of repositories maintained in `/etc/cvmfs/default.local`
+# @param cvmfs_mount_rw sets CVMFS_MOUNT_RW
+# @param cvmfs_memcache_size Sets CVMFS_MEMCACHE_SIZE in Megabytes.
+# @param cvmfs_claim_ownership Sets CVMFS_CLAIM_OWNERSHIP.
+# @param cvmfs_uid_map Specify a UID map file path for this repository . Sets CVMFS_UID_MAP
+# @param cvmfs_gid_map Specify a GID map file path for this repository . Sets CVMFS_GID_MAP
+# @param cvmfs_follow_redirects Sets CVMFS_FOLLOW_REDIRECTS.
+# @param cvmfs_external_fallback_proxy Sets CVMFS_EXTERNAL_FALLBACK_PROXY
+# @param cvmfs_external_http_proxy Sets CVMFS_EXTERNAL_HTTP_PROXY
+# @param cvmfs_external_timeout Sets CVMFS_EXTERNAL_TIMEOUT
+# @param cvmfs_external_timeout_direct Sets CVMFS_EXTERNAL_TIMEOUT_DIRECT
+# @param cvmfs_external_url Sets CVMFS_EXTERNAL_URL
+# @param cvmfs_repository_tag Sets CVMFS_REPOSITORY_TAG
+# @param mount_options Mount options to use for fstab style mounting.
+#
+define cvmfs::mount (
+  Optional[Integer]  $cvmfs_quota_limit                             = undef,
```

Comment:
Same as before. It seems that we should apply the same transform to `cvmfs_quota_limit` as `config.pp` as the `erb` refers to `@my_cvmfs_quota_limit`.